code/controllers/subsystem/job.dm

```diff
@@ -86,6 +86,14 @@ SUBSYSTEM_DEF(job)
 	Debug("AR has failed, Player: [player], Rank: [rank]")
 	return 0
 
+/datum/controller/subsystem/job/proc/FreeRole(rank)
+	if(!rank)
+		return
+	Debug("Freeing role: [rank]")
+	var/datum/job/job = GetJob(rank)
+	if(!job)
+		return FALSE
+	job.current_positions--
```

Comment:
Do we need a lower bounds check on this variable?

Reply:
Well, logic would imply we do not, so I'll include one.

Reply:
Unless some mess up happens elsewhere in the code when it comes to assigning `current_positions` there should never be a case where this underflows. - Famous last words.